dotcom-rendering/src/palette.ts

```diff
@@ -85,12 +82,49 @@ const accordionKeyEventsBackgroundLight = (): string =>
 const accordionBackgroundDark = (): string => sourcePalette.neutral[10];
 const accordionLiveFeedBackgroundLight = (): string =>
 	sourcePalette.neutral[97];
+
 const tableOfContentsLight = (): string => sourcePalette.neutral[7];
 const tableOfContentsDark = (): string => sourcePalette.neutral[86];
-
 const tableOfContentsBorderLight = (): string => sourcePalette.neutral[86];
 const tableOfContentsBorderDark = (): string => sourcePalette.neutral[20];
 
+const adLabelsTextLight = (): string => {
+	return sourcePalette.neutral[20];
```

Comment:
Looking at the [previous value](https://github.com/guardian/dotcom-rendering/pull/9316/files#diff-c640b888af8c64b465793db73a100c790a3bedbd326ee150850fa3475020705bL32) and [the web implementation](https://github.com/guardian/dotcom-rendering/blob/72d966dbe7c1a3a006c7331bb9f536e5da2bf38b/dotcom-rendering/src/components/AdSlot.web.tsx#L78), I belive this should have been `neutral[46]`.